container/BUILD

```diff
@@ -26,6 +26,71 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
+py_binary(
```

Comment:
extract_config.py has been removed from this repo as part of the migration to go container registry.